packages/dd-trace/test/plugins/externals.json

```diff
@@ -146,7 +146,7 @@
   "fastify": [
     {
       "name": "fastify",
-      "versions": ["2.15.0", "3.0.0", "3.9.2"]
```

Comment:
not sure it's a good idea to remove this. We should ask @rochdev who introduced it in the first place in this commit https://github.com/DataDog/dd-trace-js/commit/0a1befc806751bd7ae48f2b8099ef3a56ad33449

Reply:
I would say that we should not remove this. Specific versions added here are to force the CI to be executed in these versions, probably something changed in these releases and we don't want to add a regression for these versions in the future.

Reply:
@IlyasShabi what did Roch say ?